project/make/.integration-daemon-start

```diff
@@ -16,6 +16,7 @@ export DOCKER_GRAPHDRIVER=${DOCKER_GRAPHDRIVER:-vfs}
 export DOCKER_EXECDRIVER=${DOCKER_EXECDRIVER:-native}
 
 if [ -z "$DOCKER_TEST_HOST" ]; then
+	unset DOCKER_HOST
```

Comment:
I'm wondering if we should, instead of doing this, set this to something like `unix://$DEST/docker.sock` here and pass it via `--host` to the daemon we launch just below to further insulate us from the "host" we're running this daemon inside (even though in most cases right now, that's our container).